packages/services/questionnaire/src/lib/inputs/input.svelte

```diff
@@ -36,7 +36,10 @@
                 <h1 class="text-2xl">{question.title}</h1>
             {/if}
 
-            {@html question.body}
+            <p class="question-description">
```

Comment:
@html may render all html elements. Therefore, there's the possibility of rendering multiple elements which are not supposed to live inside a p tag. This shouldn't be stylised using a p tag. Possibly, you may swap it for a div but I still think this is the incorrect approach since the step is wrapped in a div [here](https://github.com/AUBGTheHUB/monolith/blob/b1ca1811bb6d9c57a208c383e72f32a3f16b32b2/packages/services/questionnaire/src/lib/inputs/input.svelte#L34C1-L35C1). Furthermore, avoid implementing custom css if it can be handled with Tailwind.